packages/idos-sdk-js/README.md

```diff
@@ -97,9 +97,13 @@ Our [`📁 idos-example-dapp`](../../apps/idos-example-dapp) shows an example of
 
 <img src="./assets/readme-container-2.gif" />
 
-### Using `hasPassword`
+### Using `hasProfile`
 
-You can check id your user has an idOS profile using await `idos.hasProfile(address)`. This can be done without a signature, unlike the `setSigner` flow described below, making your UX simpler for new users.
+You can check id your user has an idOS profile using await `idos.hasProfile(address)`. This can be done without a signature, unlike the `setSigner` flow described below, making your UX simpler for new users:
```

Comment:
```suggestion
You can check if your user has an idOS profile using await `idos.hasProfile(address)`. This can be done without a signature, unlike the `setSigner` flow described below, making your UX simpler for new users:
```